test/models/series_test.rb

```diff
@@ -79,6 +79,18 @@
       Series.where(id: [series.id, v3_series.id]).v4.must_include series
       Series.where(id: [series.id, v3_series.id]).v4.wont_include v3_series
     end
+
+    it 'searches text for title and description' do
+      series = create(:series,
+                      title: 'Some Weirdo',
+                      description: 'Unique thing',
+                      short_description: 'Lacking sense')
```

Comment:
Maybe it's the Talking Heads blaring from the basement, but this makes perfect sense to me.